src/earn/EarnPoolInfoScreen.test.tsx

```diff
@@ -56,13 +65,82 @@ describe('EarnPoolInfoScreen', () => {
       within(getByTestId('AgeCard')).getByText('duration, {"context":"month","count":5}')
     ).toBeTruthy()
 
-    expect(getByText('earnFlow.poolInfoScreen.withdraw')).toBeTruthy()
-    expect(getByText('earnFlow.poolInfoScreen.deposit')).toBeTruthy()
+    expect(
+      within(getByTestId('ActionButtons')).getByText('earnFlow.poolInfoScreen.withdraw')
+    ).toBeTruthy()
+    expect(
+      within(getByTestId('ActionButtons')).getByText('earnFlow.poolInfoScreen.deposit')
+    ).toBeTruthy()
+  })
+
+  it('renders deposit and earnings card when in pool when deposited in pool', () => {
```

Comment:
could add a test where `includedInPoolBalance` is true to test that it doesn't double count it.

Reply:
Added additional tests in e00254f5cf5b796566f5a435127bb8bbbb765c8c!